docs/admin/install/slack_setup.md

```diff
@@ -28,24 +30,207 @@ PLUGINS_CONFIG = {
+> For example, when an app named `Nautobot ChatOps` is installed to the workspace:
+>
+> 1. A message is displayed in the channel, saying that the integration`Nautobot ChatOps` has been added
+> 2. You `@Nautobot ChatOps` in the channel
+> 3. You are prompted to add `@Nautobot ChatOps` to the channel
+
+![slack integration invite](../../images/add_nautobot.png)
+
+Once these steps are completed, you can proceed to the [Install Guide](index.md#Install-Guide) section.
+
+## Configuring Multiple Chatbots in a Workspace
+
+Chatbots from multiple Nautobot implementations can exist in a single Slack workspace and even channel.
+
+They will be differentiated in the workspace using the `slack_slash_command_prefix` value in `PLUGINS_CONFIG`.
+
+Here is an example `nautobot_config.py` for the first Nautobot chatbot implementation in the workspace. This chatbot will be called in the workspace using `/nautobot`.
+
+```python
+# Enable installed plugins. Add the name of each plugin to the list.
+PLUGINS = ["nautobot_chatops"]
+
+PLUGINS_CONFIG = {
+    "nautobot_chatops": {
+        "enable_slack": True,
+        "slack_api_token": "<slack-api-token>",
+        "slack_signing_secret": "<slack-signing-secret>",
+    }
+}
+```
+
+Here is an example `nautobot_config.py` for a second Nautobot chatbot implementation in the workspace.
+This configuration explicitly configures the `slack_slash_command_prefix` key/value.
+This chatbot will be called in the workspace using `/network2-nautobot`.
+
+```python
+# Enable installed plugins. Add the name of each plugin to the list.
+PLUGINS = ["nautobot_chatops"]
+
+PLUGINS_CONFIG = {
+    "nautobot_chatops": {
+        "enable_slack": True,
+        "slack_api_token": "<slack-api-token>",
+        "slack_signing_secret": "<slack-signing-secret>",
+        "slack_slash_command_prefix": "/network2-"
+    }
+}
+```
+
+> NOTE: by default, your slash command must end with `nautobot`. If no `slack_slash_command_prefix` is specified,
+> the slash command will be `/nautobot`. If a `slack_slash_command_prefix` is specified, the slash command will be `<slack_slash_command_prefix>nautobot`.
+
+> NOTE: Custom chatbot development allows for chatbot slash commands such as `/grafana` and `/meraki`.
+
+## General Chat Setup Instructions
+
+See [admin_install](index.md) instructions here for general plugin setup instructions.
+
+## Startup Slack Sockets (Socket Mode)
+
+Socket Mode requires it's own Worker in order poll Slack for new slash commands and interactions. Slack Socket Mode supports up to 10 active connections. To start a new connection there is a new Management Command provided.
+
+### start_slack_socket Management Command
+
+`nautobot-server start_slack_socket`
+
+Start a worker to connect to Slack.
+
+### Slack Socket Systemd Service
+
+To establish the `systemd` unit file for the Slack Socket worker, copy and paste the following into `/etc/systemd/system/chatops-slack-socket.service`:
```

Comment:
Feel like is should have nautobot in the service file name similar to the worker

Reply:
Thanks! I was trying to keep the name short, but I agree this is better.